packages/aragon-wrapper/src/interfaces.js

```diff
@@ -20,15 +27,66 @@ const ABIS = {
   'aragon/Forwarder': abiAragonForwarder.abi,
   'aragon/Kernel': abiAragonKernel.abi,
   'aragon/EVM Script Registry': abiAragonEVMScriptRegistry.abi,
-  'aragon/Repo': abiAragonRepo.abi,
+  'apm/APM Registry': abiApmRegistry.abi,
+  'apm/Repo': abiApmRepo.abi,
+  'apm/ENS Subdomain Registrar': abiApmEnsSubdomainRegistrar.abi,
   'standard/ERC20': abiERC20.abi
 }
 
 const ARTIFACTS = {
   'aragon/ACL': artifactsAragonACL,
   'aragon/Kernel': artifactsAragonKernel,
-  'aragon/EVM Script Registry': artifactsAragonEVMScriptRegistry
+  'aragon/EVM Script Registry': artifactsAragonEVMScriptRegistry,
+  'apm/APM Registry': artifactsApmRegistry,
+  'apm/Repo': artifactsApmRepo,
+  'apm/ENS Subdomain Registrar': artifactsApmEnsSubdomainRegistrar
 }
 
+const SYSTEM_APP_MAPPINGS = new Map([
+  [apmAppId('acl'), 'ACL'],
+  [apmAppId('evmreg'), 'EVM Script Registry'],
+  [apmAppId('kernel'), 'Kernel']
+])
+
+const APM_APP_MAPPINGS = new Map([
+  [apmAppId('apm-registry'), 'APM Registry'],
+  [apmAppId('apm-repo'), 'Repo'],
+  [apmAppId('apm-enssub'), 'ENS Subdomain Registrar'],
+  // Support open.aragonpm.eth's native packages
+  // Note that these were erroneously deployed on the open.aragonpm.eth instance rather than
+  // reusing the aragonpm.eth versions
+  [apmAppId('apm-registry.open'), 'APM Registry'],
+  [apmAppId('apm-repo.open'), 'Repo'],
+  [apmAppId('apm-enssub.open'), 'ENS Subdomain Registrar']
+])
+
+const APP_NAMESPACE_MAPPINGS = new Map([
+  ['aragon', SYSTEM_APP_MAPPINGS],
+  ['apm', APM_APP_MAPPINGS]
+])
+
 export const getAbi = name => ABIS[name] || null
 export const getArtifact = name => ARTIFACTS[name] || null
+
+export function getAppInfo (appId, namespace) {
+  const mapping = APP_NAMESPACE_MAPPINGS.get(namespace)
+
+  if (!mapping || !mapping.has(appId)) {
+    return null
+  }
+
+  const appName = mapping.get(appId)
```

Comment:
It seems that `appName` is in fact two different types, used in two incompatible ways. 

1. As an ENS name, e.g. `apm-registry.aragonpm.eth`. Has 1:1 relationship with the `appID`(just a hash of it)
1. As a readable case sensitive identifier, e.g. `APM Registry`

The second use case for this type is more general because a single `appName` can have multiple instances. What do you think about indicating this by renaming from `appName`?

Reply:
Good point; we generally use `name` instead for the human readable name so let's use that here too.